Web/src/components/Homepage.tsx

```diff
@@ -71,7 +55,7 @@ export function Homepage() {
         </Grid>
         <Grid item xs={12}>
           <BowlpoolTable
-            year={2022}
+            year={2023}
```

Comment:
maybe put in env

Reply:
created a jira: [BP-87](https://rskennedy.atlassian.net/browse/BP-87)